packages/mask/src/plugins/Savings/SNSAdaptor/SavingsTable.tsx

```diff
@@ -129,11 +129,14 @@ export function SavingsTable({ chainId, tab, mappableProtocols, setSelectedProto
                             </Grid>
                             <Grid item xs={3} className={classes.tableCell}>
                                 <Typography variant="body1">
-                                    {Number(protocol.balance) < 0.000001
-                                        ? '<0.000001'
-                                        : Number(protocol.balance).toLocaleString(undefined, {
-                                              maximumFractionDigits: 6,
-                                          })}
+                                    {Number(protocol.balance) !== 0
+                                        ? Number(protocol.balance) < 0.000001
+                                            ? '< 0.000001'
+                                            : Number(protocol.balance).toLocaleString(undefined, {
+                                                  minimumFractionDigits: 6,
+                                                  maximumFractionDigits: 6,
+                                              })
+                                        : 0}
```

Comment:
plz this expression extract as function

Reply:
@septs, I've updated it to:

```
const formattedBalance = formatBalance(balance, 18, 6)
```

That's my bad!